src/sections/UpcomingEventsSection.js

```diff
@@ -1,17 +1,86 @@
+/* eslint-disable operator-linebreak */
+/* eslint-disable function-paren-newline */
+/* eslint-disable implicit-arrow-linebreak */
```

Comment:
Ideally we don't want to disable the eslint unless its absolutely necessary

Reply:
the formatter was changing long lines into multiple lines which causes errors with eslint bc it didn't like those linebreaks. i'm not sure how to fix this but i moved the eslint disable commands to only do the specific lines instead of the whole file

Reply:
yeah the formatter isn't perfect rn :( thats works